torch/csrc/jit/codegen/cuda/lower_unroll.cpp

```diff
@@ -176,25 +191,22 @@ kir::ForLoop* handleMisalignedVectorization(
   new_loop->body().push_back(pred_ite);
 
   // Generate vectorize index
-  auto index = (out_tv->memoryType() == MemoryType::Global)
-      ? Index::getConsumerIndex(in_tv->fuserTv(), loop_structure)
-      : Index::getProducerIndex(
+  auto indices = (out_tv->memoryType() == MemoryType::Global)
+      ? Index::getConsumerStridedIndices(out_tv->fuserTv(), loop_structure)
+      : Index::getProducerStridedIndices(
             in_tv->fuserTv(), out_tv->fuserTv(), loop_structure);
 
   //  Get full extent for merged root domains
-  auto extent = getVectorizeExtent(index);
+  auto extent = getVectorizeExtent(vec_tv, indices);
 
   auto vector_size =
       vec_tv->domain()->domain().back()->extent()->as<kir::Int>();
 
   // Disable vectorize flag in child For-Loop
   loop_structure.back()->setVectorize(false);
 
-  auto address = ir_builder.create<kir::TensorIndex>(
-      index->view()->fuserTv(),
-      index->indices(),
-      index->rootDomainIndexMap(),
-      true);
+  auto address =
```

Comment:
Don't we want the address of the global memory tensor? I think `non_vec_tv` is the non-global memory tv.

Reply:
Ah. Maybe `non_vec_tv` should be `global_tv`, and `vec_tv` => `local_vec_tv`